chrome/content/zotero/fileInterface.js

```diff
@@ -510,14 +510,17 @@ var Zotero_File_Interface = new function() {
 			format = "rtf";
 		}
 		
+		// determine locale preference
+		var locale = io.locale;
+		
 		// generate bibliography
 		try {
 			if(io.method == 'copy-to-clipboard') {
-				copyItemsToClipboard(items, io.style, false, io.mode === "citations");
+				copyItemsToClipboard(items, io.style, locale, false, io.mode === "citations");
```

Comment:
This is a public method, so it should be called with `this.`. (If `copyItemsToClipboard` used `this`, it would break when called from here. This is why we no longer use the top-of-file function declarations — the `this.foo = function () {...}` format prevents accidental private calls like this.)